package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225509546-e46147ef-bcbe-448c-a475-78eed7844014.png)

[OPTIONAL]

- [ ] It would be nice if you add the following commands in their respective sections to make your readme file more descriptive and professional 😊

### Install
```
cd metrics
npm install
```
### Usage
```
npm start
```
### Test
```
npm test
```
### Deployment
```
npm run build
```